core/src/main/scala/sri/core/AppConfig.scala

```diff
@@ -0,0 +1,11 @@
+package sri.core
+
+import scala.scalajs.js
+import scala.scalajs.js.UndefOr
+
+@js.native
+trait AppConfig extends js.Object {
+  def appKey: String = js.native
+  def component: UndefOr[js.Function0[ReactClass]] = js.native
+  def run: UndefOr[js.Function]
+}
```

Comment:
https://github.com/facebook/react-native/blob/efcdef711eba82b2905d237ee9a3d094652c37ac/Libraries/ReactNative/AppRegistry.js#L39

AppConfig should be @ScalajsDefined , and its part of react-native so it should be under mobile module.